gui/src/services/api-register.ts

```diff
@@ -69,6 +69,7 @@ export default {
   health_components: `api/${version}/system/health/components`,
   system_status: `/api/${version}/system/status`,
   health_resources: `api/${version}/system/health/resources`,
+  health_cluster: `api/${version}/system/health/node`,
```

Comment:
No need of hard coding node here. It is only required at GUI. You can pass the value that you are getting from gui here.

Reply:
Added the comment by mistake